src/popup/pages/SignMessage.vue

```diff
@@ -19,10 +20,12 @@ export default defineComponent({
       try {
         const aeSdk = await getAeSdk();
         const message = route.query.message?.toString();
+        const isHexEncodedMessage = message && route.query.encoding?.toString() === 'hex';
+        const displayMessage = isHexEncodedMessage ? hexToUtf8(message) : message;
```

Comment:
```suggestion
        const displayMessage = isHexEncodedMessage ? Buffer.from(message, 'hex').toString() : message;
```

Reply:
I've applied this change
@AndreasGassmann  please note that this means that values need to be cleaned from the `0x` prefix now

Reply:
`0x` prefix is not needed because it is already stated in `encoding=hex`, also we can't detect `hex` by the presence of `0x` prefix because aepp may want to sign such a message as it is.